src/components/include/utils/threads/thread_delegate.h

```diff
@@ -45,7 +48,12 @@ class Thread;
  * Thread procedure interface.
  * Look for "threads/thread.h" for example
  */
+#ifdef QT_PORT
+class ThreadDelegate : public QObject {
```

Comment:
Necessary investigate in  [SDLWIN-308](https://adc.luxoft.com/jira/browse/SDLWIN-308)